README.md

```diff
@@ -84,19 +98,21 @@ tested with the default version of Perl in the OSs as listed in the table above.
 
 ### Supported Client Browser versions
 
-Zonemaster GUI is tested against the browsers, their versions and listed OS as indicated bellow and should work perfectly with similar configurations.
+Zonemaster GUI is tested against the browsers, their versions and listed OS as
+indicated bellow and should work perfectly with similar configurations.
 
 Operating System | Browser | Version
 ---------------- | ------- | -------
-Ubuntu 18.04     | Firefox | 64, 65, 66
-Ubuntu 18.04     | Chrome  | 66
-Windows 10       | Firefox | 64, 65, 66
-Windows 10       | Chrome  | 73
-MacOs            | Firefox | 65
-MacOs            | Chrome  | 73
+Ubuntu 18.04     | Firefox | 69
```

Comment:
Why did we have several versions before?

Reply:
I do not know. Now I used the information from the Browser vendors.